src/imitation/scripts/ingredients/demonstrations.py

```diff
@@ -109,43 +141,15 @@ def generate_expert_trajs(
         )
 
 
-@demonstrations_ingredient.capture
-def load_local_expert_trajs(
-    rollout_path: Union[str, pathlib.Path],
-    n_expert_demos: Optional[int],
-) -> Sequence[types.Trajectory]:
-    """Loads expert demonstrations from a local path.
-
-    Args:
-        rollout_path: A path containing a pickled sequence of `types.Trajectory`.
-        n_expert_demos: The number of trajectories to load.
-            Dataset is truncated to this length if specified.
-
-    Returns:
-        The expert trajectories.
-
-    Raises:
-        ValueError: There are fewer trajectories than `n_expert_demos`.
-    """
-    expert_trajs = serialize.load(rollout_path)
-    logger.info(f"Loaded {len(expert_trajs)} expert trajectories from '{rollout_path}'")
-    if n_expert_demos is not None:
-        if len(expert_trajs) < n_expert_demos:
-            raise ValueError(
-                f"Want to use n_expert_demos={n_expert_demos} trajectories, but only "
-                f"{len(expert_trajs)} are available via {rollout_path}.",
-            )
-        expert_trajs = expert_trajs[:n_expert_demos]
-        logger.info(f"Truncated to {n_expert_demos} expert trajectories")
-    return expert_trajs
-
-
 @demonstrations_ingredient.capture(prefix="expert")
-def _download_expert_rollouts(rollout_type, loader_kwargs):
+def _download_expert_rollouts(rollout_type: str, loader_kwargs: Dict[str, Any]):
     assert rollout_type.endswith("-huggingface")
     algo_name = rollout_type.split("-")[0]
-    return serialize.load_rollouts_from_huggingface(
+    model_name = hfsb3.ModelName(
         algo_name,
-        env_name=loader_kwargs["env_name"],
-        organization=loader_kwargs["organization"],
+        hfsb3.EnvironmentName(loader_kwargs["env_name"]),
     )
+    repo_id = hfsb3.ModelRepoId(loader_kwargs["organization"], model_name)
+    logger.info(f"Loading expert trajectories from {repo_id}")
+    dataset = datasets.load_dataset(repo_id)["train"]
```

Comment:
Having canonical splits for the datasets might also be useful, but that might be beyond the scope of this PR.

Reply:
Turns out I could add this feature easily as a side-effect (see comment above)